### ISSUE TITLE
Some minor tweaks to the Domain documentation

### ISSUE THREAD

**Issue by mikelittle** [humanmade/altis-cloud#759]
I have a couple of suggestions for improvement for the  [domain documentation page](File: https://github.com/humanmade/altis-cloud/blob/master/docs/dns-configuration.md)

Line 26: (This is not the same as domain registration, and Altis does not act as your registrar. Altis does not currently offer the ability to purchase new domains.)

Maybe clarify the use of "registration" some more. Perhaps "(This is not the same as domain registration, that is, buying a domain, ..."
OR we can use a different word instead of "registration"? It really is a pre-loaded term. Perhaps "Configure"? e.g. "Configure a domain with the CDN"?

Line 32: Enter your base domain, excluding any `www.` or wildcards.

Technically, this can also be a hostname rather than a ~`tld`~ domain - e.g. shop.example.com when you don't already have the wildcard. Should we mention that?

Line 64: The final step is to route your custom domain to your Altis environment. This is done by adding a `CNAME` DNS record for your domain or subdomains.

Do we need to say this should be in place of any A records and/or any A/AAAA records should be removed.

Line 81: The Dashboard will show you your project-specific domain to use in CNAME record

"...project-specific domain to use in CNAME records." (plural)

**Comment by rmccue** [comment]
> Maybe clarify the use of "registration" some more. Perhaps "(This is not the same as domain registration, that is, buying a domain, ..."
> OR we can use a different word instead of "registration"? It really is a pre-loaded term. Perhaps "Configure"? e.g. "Configure a domain with the CDN"?

Yeah, it's not the greatest verb, but I left it basically as it already was for now (I did feel like it was worth clarifying since I was already in there updating stuff).

It's debatable whether it's even really a step now from the user's perspective, since it's effectively now just "filling out a form"; maybe we could eliminate it altogether?

> Technically, this can also be a hostname rather than a `tld` - e.g. shop.example.com when you don't already have the wildcard. Should we mention that?

Well, the TLD would be `com` there, which is why it says "domain" rather than TLD. Maybe "If you're adding an extra subdomain, enter the full subdomain." or something? Tough to be clear yet concise here.

> Do we need to say this should be in place of any A records and/or any A/AAAA records should be removed.

CNAME will override any A records anyway, but that's on the DNS provider IMO to sort out. I went light on our advice here (unlike consumer-facing hosts typically) since I'm assuming people know how to add DNS records.
